api/v3/openapi.yaml

```diff
@@ -1561,6 +1561,7 @@ components:
             - is_internal
             - replication_factor
             - partitions_count
+            - authorized_operations
```

Comment:
It's not required, right? If you don't set `include_authorized_operations=true`, you won't get this property.